src/main/python/hydra/lib/hydrabase.py

```diff
@@ -38,24 +38,26 @@ def debug(sig, frame):
 class AppGroup(object):
     """
     Class to hold info about an APP group.
-    Allows cabilities to execute methods on the analyser
+    Allows capabilities to execute methods on the analyser
     as passed by the caller
-    @args:
-    hydra:          hydra handle
-    app_name:       App name
-    group_name:     group name
-    analyser:       Analyser class "name" e-g HAnalyser not HAnalyser()
     """
-    def __init__(self, hydra, app_name, group_name, num_app_instances, analyser=None):
-        self.group_info = {}
+    def __init__(self, hydra, app_name, group_name, num_tasks, analyser=None):
+        """
+        :param hydra: Hydra handle
```

Comment:
probably a nitpick, but the existing convention we use in code for argument description is like
@args:
arg    :  description 

Check some other functions for example.